src/Support/Support.Autorest/README.md

```diff
@@ -60,39 +57,83 @@ directive:
     set:
       format-table:
         properties:
-          # - Id
           - DisplayName
           - Name
-          - SecondaryConsentEnabled # Do we need this? 
-          # - Type
+          - SecondaryConsentEnabled
   - where:
       model-name: Service
     set:
       format-table:
         properties:
-          # - Id
           - DisplayName
           - Name
-          - ResourceType # Do we need this? 
-          # - Type
+          - ResourceType
   - where:
       model-name: FileDetails
     set:
       format-table:
         properties:
-          # - Id
           - Name
           - CreatedOn
           - ChunkSize
           - FileSize
           - NumberOfChunks 
-          # - Type
+  - where:
+      subject: UploadFile
+      parameter-name: FileWorkspaceName
+    set:
+      alias: WorkspaceName
+  - where:
+      subject: UploadFilesNoSubscription
+      parameter-name: FileWorkspaceName
+    set:
+      alias: WorkspaceName
+  - where:
+      subject: FileWorkspacesNoSubscription
+      parameter-name: FileWorkspaceName
+    set:
+      alias: Name
```

Comment:
why is there another alias 'Name' instead of 'WorkspaceName' here?

Reply:
Autorest automatically renames parameters when they correlate to the subject of the cmdlet (e.g. aliases FileWorkspaceName for AzSupportFileWorkspace cmdlets to Name and FileWorkspaceName for AzSupportFile cmdlets to WorkspaceName). However, it doesn't recognize the NoSubscription versions of the cmdlets as the same subject. Without that part of the readme, AzSupportFileWorkspacesNoSubscription doesn't recognize Name as an alias for FileWorkspaceName. However, now I'm wondering if we should try to keep the parameter names consistent between all cmdlets (e.g. keep FileWorkspaceName as the parameter name for all cmdlets). Let's check with Anagh